addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -61,6 +61,15 @@ const LinkPopoverWidget = Widget.extend({
         .popover('show')
         .data('bs.popover').tip.classList.add('o_edit_menu_popover');
 
+        // hide popover while typing on mega menu
+        if (this.$target.parents('.o_mega_menu').length) {
```

Comment:
Why only for the mega menu? I don't see the reason. And if we really want to do that, then this code should be in website, not web_editor :wink:

Reply:
As per the task specs (which we discussed with SBU).
Because the behavior seems weird only in the mega menu where it seems a bit messy/overcrowded.
See 
![image](https://user-images.githubusercontent.com/30048408/122078345-ce358a00-cdfc-11eb-9466-b7cb1385792d.png)
I don't see the point to hide it anywhere else as it doesn't seems messy except in the mega menu.
Google Docs doesn't hide it when typing, but it is a bit different since you can close it on Gdoc so the comparison is unfair.

> And if we really want to do that, then this code should be in website, not web_editor wink

No :) in website this is only for `menu_data` elements, so the regular menus.
Mega menu are considered/handled as non `menu_data`/regular page content

Reply:
Oh, I don't mind switching it to every links by the way, I just think it is better to not hiding it when typing (no hide/show/flickering + no good reason to hide it?), but I can live with both :)

Reply:
It just feels exactly the same to me in the mega menu or not. I don't see the messy stuff ^^ So I don't really care if we hide the popover or not but I don't see why this should be different in mega menu only :wink: Anyways, if @sbuhl has a reason it's fine by me :+1: 

> No :) in website this is only for menu_data elements, so the regular menus.
> Mega menu are considered/handled as non menu_data/regular page content

That does not change the fact that the "o_mega_menu" class is not supposed to be known in the web_editor app and should thus not appear in its folder :wink: Probably something to do like a function which returns nothing in web_editor and override that function in website.

Reply:
> I don't see the messy stuff ^^

I don't either, and I wouldn't have hidden it at all. But after digging a bit, it seems like it is the mega menu that looks weird only, that's why I restricted there only.
That way, 99% of the users won't have the behavior (as they probably don't have a mega menu).

> So I don't really care if we hide the popover or not 

I don't really care too, I just prefer **a bit** to not hide it.

Anything will do to me, little detail I don't really mind that much.
SBU had to confirm it was ok to target only mega menu (see the task), he was waiting for the PR to have final opinion.